scripts/kubestellar

```diff
@@ -39,9 +39,93 @@ cleanup=0
 espw_name="espw"
 log_folder=$(pwd)/kubestellar-logs
 local_kcp=true
+imws="root:imw1"
+wmws="root:wmw1"
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
+
+function echoerr() {
+   echo "ERROR: $1" >&2
+}
+
+
+function ensure_imw() {
+    while IFS=':' read -ra items; do
+        for i in "${items[@]}"; do
+            if [ "$i" == "" ] ; then
+                break
+            fi
+            if ! kubectl ws "$i" &> /dev/null ; then #
+                if ! kubectl ws create "$i" --enter &> /dev/null ; then
+                    echoerr 'unable to create "$i"!'
+                    break
+                fi
+            fi
+        done
+    done <<< "$1"
+    if ! kubectl get apibinding "edge.kubestellar.io" &> /dev/null; then
+    	kubectl kcp bind apiexport root:espw:edge.kubestellar.io
+    fi
+}
+
+function ensure_imws() {
+    initial_ws=$(kubectl ws . --short)
+    while IFS=',' read -ra items; do
+        for i in "${items[@]}"; do
+            if [ "$i" == "" ] ; then
+                continue
+            fi
+            echo "Ensuring INV: $i"
```

Comment:
The old name is "IMW" (Inventory Management Workspace), the new name is "IS" (Inventory Space).

Reply:
@MikeSpreitzer Thank you for the observation. I will change to IMW as it it used in a lot of spaces.
I think I need a bigger more coordinated and, breaking, intervention to change all of IMW into IS.